### ISSUE TITLE
Should HP:0012071 be "Abnormal circulating acylcarnitine concentration"?

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#7621]
**HPO term**
HP:0012071

**Bug description**

It seems there is conflation of the general term acylcarnitine and the specific compound acetylcarnitine, the C2 form (see Mayo listing at https://neurology.testcatalog.org/show/ACRN ).  It seems that HP:0012071 should be "Abnormal circulating acylcarnitine concentration".  Under HP:0012071 there is HP:0045045
 "Elevated circulating acylcarnitine concentration", where the correct term is used.  However, its child terms correctly include HP:0033443 "Elevated circulating propionylcarnitine concentration" for the C3 form, but no an entry for acetylcarnitine, the C2 form, presumably because the typo in HP:0012071 made it seem as if that was the C2 entry.

In addition, it would add to the clarity to include the C terminology (e.g., C2 or C3 or C4-OH) as used in the Mayo listings.

**Comment by pnrobinson** [comment]
@MickeySegal  not sure about this
HP:0012071 **IS** ["Abnormal circulating acylcarnitine concentration"](https://hpo.jax.org/app/browse/term/HP:0012071)
HP:0045045 IS [Elevated circulating acylcarnitine concentration ](https://hpo.jax.org/app/browse/term/HP:0045045)

However I agree with the comment about adding the C terminology. Could you please clarify

**Comment by MickeySegal** [comment]
At least on the public version at https://hpo.jax.org/app/browse/term/HP:0012071, the title is "Abnormal circulating acetylcarnitine concentration", as if it were the C2 form, but it seems to intend to be the category term acylcarnitine, as one understands from reading the description in italics below the title.  Also there are no entries meant to be for acetylcarnitine (C2), which should be added both for high and low abnormalities, each of which is observed in disease states.

On the C terminology, some but not all of the HPO entries use the C terminology  (e.g., C2 or C3 or C4-OH) as on https://neurology.testcatalog.org/show/ACRN, which is particularly important since some of the terms, not just the ones discussed here, look so similar that one needs to stare at them for a while to see that they are different.

**Comment by pnrobinson** [comment]
@MickeySegal 

Ups, there was a spelling error -- you are correct, I changed the label to 
Abnormal circulating acylcarnitine concentration

I am also reorganizing the hierarchy to take the C nomenclature into account.
